.circleci/config.yml

```diff
@@ -400,6 +405,8 @@ workflows:
       - build:
           name: build-rook
           container-image: rook
+          requires:
+            - build-ceph-container
```

Comment:
This seems not necessary, because uploading to `quay.io` happens only in the `master` branch.

Reply:
I understood. Applied.